Solutions/Marain.TenantManagement.Specs/Features/ServiceManifests/Validation.feature

```diff
@@ -0,0 +1,74 @@
+﻿@perScenarioContainer
+@useInMemoryTenantProvider
+
+Feature: Service manifest validation
+	In order to maintain the integrity of the system
+	As a developer
+	I want to be able to validate Service Manifests
+
+Background: 
+	Given the tenancy provider has been initialised for use with Marain
+
+Scenario: Manifest name not set
+	Given I have a service manifest called 'Manifest' with no service name
+	When I validate the service manifest called 'Manifest'
+	Then an 'InvalidServiceManifestException' is thrown
+	
+Scenario Outline: Creating a manifest with invalid service names
+	Given I have a service manifest called 'Manifest' for a service called '<Service Name>'
+	When I validate the service manifest called 'Manifest'
+	Then an 'InvalidServiceManifestException' is thrown
```

Comment:
Do we care about the message? Is it a goal here for the tooling to tell me what I've done wrong? (I know the `Feature` definition at the top doesn't say that, but better descriptions of misconfigurations was a broad goal for this work, right?)